src/pipelines/zero_shot_classification.rs

```diff
@@ -627,8 +627,8 @@ impl ZeroShotClassificationModel {
         let max_len = tokenized_input
             .iter()
             .map(|input| input.token_ids.len())
-            .max()
-            .unwrap();
```

Comment:
You could turn the `None` into an `Error` here using `or_or_else` - as this is invalid for the rest of the pipeline (there are no handlers for `None` for the caller of this method. This would help producing more accurate error message (for example in this case the issue is that either the input text or the labels are an empty list).